utils/query.test.js

```diff
@@ -0,0 +1,41 @@
+const assert = require('assert');
```

Comment:
If you append `.strict` you can simply the calls named `assert.strictSomething`.

Reply:
Oh wow, that's convenient! Fixed with 10032cb12df7815ce60537a2dbfdd4e3fab448c7.